src/main/java/edu/uc/javahotkey/JavaHotKeyController.java

```diff
@@ -17,17 +17,16 @@ public String index() {
         return "index";
     }
 
-    @GetMapping("/sampleJsonSchema")
+    @GetMapping("/GetHotKeys{keyMap}")
+    public ResponseEntity fetchAllHotKeys(){return new ResponseEntity(HttpStatus.OK);}
     @ResponseBody
     public Project sampleJsonSchema() {
         Project project = new Project();
         project.setLua("function func_name(keys)\\nDown(Key.A)\\nSleep(25)--ms\\nUp(Key.A)\\nPress(Key.B)\\nen");
-        project.setName("Sample Project");
+        project.setName("Java Hot-Keys");
         KeyMap keyMap = new KeyMap();
         keyMap.setFunctionName("func_name");
         keyMap.setKeymap(new ArrayList<Integer>());
-        keyMap.getKeymap().add(17);
-        keyMap.getKeymap().add(65);
```

Comment:
Removing this hardcoded data because it simply does not belong here and needs to go into a stub/service.